helm/idp/keycloak-init.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+# Installs all idp keycloak-init
+## Usage: ./keycloak-init.sh [kubeconfig]
+
+if [ $# -ge 1 ] ; then
+  export KUBECONFIG=$1
+fi
+
+NS=idp
+CHART_VERSION=12.0.1-B2
```

Comment:
Version?

Reply:
This version is keycloak-init helm chart version